jnfang/brisk-learning
# Add postgres DB to app with initial users and user_tokens table

Summary:
Adds the ability to connect to a postgres DB, initalize the schema
for it.  Also adds our first use of app secrets and consolidates
how we access the single for the DB and the for app.

https://app.asana.com/0/1203963408726580/1203963408726599

Test Plan
1. Fill in .env file with dev DB address
2. Run python3 models.py
3. Confirm schema is created and I can add and query data
4. Run python3 main.py and confirm it runs as normal
